RepoCleanup/Functions/SharedFunctionSnippets.cs

```diff
@@ -71,6 +71,80 @@ public static bool ShouldThisApplyToAllOrgs()
             return YesNo("Should this apply to all organisations?");
         }
 
+        public static Organisation CollectNewOrgInfo()
+        {
+            var shortName = CollectShortNameForOrg();
+            var fullname = CollectFullNameForOrg();
+            var website = CollectWebSiteForOrg();
+
+            var org = new Organisation
+            {
+                Username = shortName,
+                Fullname = fullname,
+                Website = website,
+                Visibility = "public",
+                RepoAdminChangeTeamAccess = false
+            };
+
+            return org;
+        }
+
+        private static string CollectWebSiteForOrg()
+        {
+            var isValid = false;
+            var website = string.Empty;
+
+            while (!isValid)
+            {
+                Console.Write("\r\nSet website for org: ");
+                website = Console.ReadLine();
+
+                if (string.IsNullOrEmpty(website))
+                {
+                    isValid = true;
+                }
+                else
+                {
+                    isValid = Regex.IsMatch(website, "^[a-zA-Z0-9\\-._/:]*$");
```

Comment:
Should we allow Norwegian letters?

Reply:
I know it's allowed on host names, but is it recommended? I haven't changed this part, just moved it, so it's always been like this. Perhaps consider it if it becomes a issue?